src/main/webapp/js/finscholar/homepagecontroller/homepagecontroller.js

```diff
@@ -31,22 +32,12 @@ const {PageController} = goog.require('pagecontroller');
 class HomePageController extends PageController {
   constructor() {
     super();
-    this.eventContract_ = new JsactionEventContract();
-    // Events will be handled for all elements under this container.
-    this.eventContract_.addContainer(
-        /** @type {!Element} */ (GoogDom.getElement('main')));
-    // Register the event types we care about.
-    this.eventContract_.addEvent('click');
-    this.dispatcher_ = new JsactionDispatcher();
-    this.eventContract_.dispatchTo(
-        this.dispatcher_.dispatch.bind(this.dispatcher_));
-    this.dispatcher_.registerHandlers(
-        'homepagecontroller',  // the namespace
-        null,                  // handler object
-        {
-          // action map
-          'clickAction': this.doStuff,
-        });
+    /**
+     * @private @constant
+     * @type {!ScholarshipPageView} 
```

Comment:
I don't think you need `@type` if you're making this constant.

`@private @const {!ScholarshipPageView}`